diff --git a/Makefile.am b/Makefile.am
index 7e15851..840946d 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -346,6 +346,9 @@ pspell_include_HEADERS = interfaces/cc/pspell.h
 
 endif
 
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = aspell.pc
+
 ########################################################################
 #
 # Misc Top level
diff --git a/configure.ac b/configure.ac
index 60e3b39..99f45db 100644
--- a/configure.ac
+++ b/configure.ac
@@ -589,5 +589,6 @@ AC_CONFIG_FILES([Makefile gen/Makefile common/Makefile lib/Makefile data/Makefil
 		 scripts/Makefile examples/Makefile
 		 prog/Makefile manual/Makefile po/Makefile.in m4/Makefile
                  modules/filter/Makefile myspell/Makefile lib5/Makefile
+                 aspell.pc
                  ])
 AC_OUTPUT
diff --git a/scripts/mkconfig b/scripts/mkconfig
index 608e3f7..f15f31c 100755
--- a/scripts/mkconfig
+++ b/scripts/mkconfig
@@ -15,7 +15,7 @@ case \$1 in
   echo "$2"
   ;;
 --pkgdatadir | pkgdatadir)
-  echo "$3"
+  pkg-config aspell --variable=pkgdatadir
   ;;
 *)
   echo "usage: pspell-config version|datadir|pkgdatadir"
--- /dev/null	2007-01-02 09:09:01.616000852 +0100
+++ aspell-0.60.5/aspell.pc.in	2007-01-02 14:59:04.000000000 +0100
@@ -0,0 +1,12 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+pkgdatadir=@pkgdatadir@
+
+Name: Aspell
+Description: A spelling checker.
+Version: @VERSION@
+Requires: 
+Libs: -L${libdir} -laspell
+Cflags: -I${includedir}
